src/Cache.php

```diff
@@ -51,8 +51,13 @@ public function init() {
 			$this->cli->register_command( 'spinupwp cache', CacheCommands::class );
 		}
 
+		if ( $this->is_page_cache_enabled() ) {
```

Comment:
@leeshadle We'll want to scope the display of this menu item more specifically as it's currently showing on the admin dashboard. Do you think it would make sense to suppress this on any admin page?